.gitignore

```diff
@@ -1,3 +1,4 @@
+.vagrant
```

Comment:
I'm not sure about it, It's generally should be putted on your `gitignore global`